lipnet/lipreading/videos.py

```diff
@@ -4,6 +4,7 @@
 from scipy import ndimage
 from scipy.misc import imresize
 import skvideo.io
+import cv2
```

Comment:
Handle cv2 import failure (when users don't have OpenCV) as we want to give options to users to opt-out from using the feature

Reply:
I've decided to import cv2 later in the code